api/api/models/__init__.py

```diff
@@ -128,12 +130,26 @@ class Meta:
     creation_date = models.DateTimeField(auto_now=True, editable=False)
 
 
-AuthenticatedUser = Pentester | Admin
+class Team(models.Model):
+    """Model for representing a team of pentester, with one manager"""
+    class Meta:
+        verbose_name = 'Team'
+        verbose_name_plural = 'Teams'
+        ordering = ['name']
+
+    REQUIRED_FIELDS = ['name', 'owner', 'members']
+
+    name: models.CharField = models.CharField(max_length=32)
+    owner: Manager = models.OneToOneField(Manager, on_delete=CASCADE)
```

Comment:
aren't "leader" or "manager" some better names ?

Reply:
i agree with this change